android-app/build.gradle

```diff
@@ -1,5 +1,5 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '7.3.1' apply false
-    id 'com.android.library' version '7.3.1' apply false
+    id 'com.android.application' version '7.4.2' apply false
+    id 'com.android.library' version '7.4.2' apply false
 }
```

Comment:
Please revert this for easier development on multiple branches. I will put a PR soon where I update the version to 8.0 but for now it should stay consitent across every branch because a gradle sync is necessary when swapping to this branch